R/stat-.r

```diff
@@ -122,7 +127,20 @@ Stat <- ggproto("Stat",
       )
     }, stats, groups, SIMPLIFY = FALSE)
 
-    rbind_dfs(stats)
+    data_new <- rbind_dfs(stats)
```

Comment:
`rbind_dfs()` will be removed in favour of `vec_rbind()` soon, just to remember depending on whether this gets merged before or after

Reply:
I just saw your PR. I think I'll wait until you've merged it and then apply the update in my code.